spec/cypress/app_commands/scenarios/current_privacy_policy.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+FactoryBot.create :privacy_policy
```

Comment:
Ehh, can we do it with the actual current privacy policy? 

I don't really care if the lorem ipsum paragraphs are accessible 😂

Reply:
Ah, ignore me, we publish fresh version below that. My bad.

Reply:
Well, the only reason to create this policy is because the Publish service requires previous version to be present. It will not be displayed in test.